common/persistence/visibility/store/sql/query_converter_sqlite.go

```diff
@@ -307,3 +310,26 @@ func (c *sqliteQueryConverter) buildFtsSelectStmt(
 		),
 	}
 }
+
+func (c *sqliteQueryConverter) buildCountStmt(
+	namespaceID namespace.ID,
+	queryString string,
+) (string, []any) {
+	var whereClauses []string
+	var queryArgs []any
+
+	whereClauses = append(
+		whereClauses,
+		fmt.Sprintf("(%s = ?)", searchattribute.GetSqlDbColName(searchattribute.NamespaceID)),
+	)
+	queryArgs = append(queryArgs, namespaceID.String())
+
+	if len(queryString) > 0 {
+		whereClauses = append(whereClauses, fmt.Sprintf("(%s)", queryString))
+	}
+
+	return fmt.Sprintf(
+		"SELECT COUNT(1) FROM executions_visibility WHERE %s",
```

Comment:
Had to use ChatGPT to explain me what is `COUNT(1)` and why it is better than `COUNT(*)`.